abstract.js

```diff
@@ -5,6 +5,13 @@ const pump = require('pump')
 const through = require('through2')
 const Packet = require('aedes-packet')
 
+function orderedKeys (inputObj) {
+  return Object.keys(inputObj).sort().reduce((obj, key) => {
+    obj[key] = inputObj[key]
+    return obj
+  }, {})
+}
```

Comment:
LOL that's so tricky :confused: I have all aedes repos with tape to update but with version 5 I think there is definetly something that isn't working... @mcollina thoughts on this? I also tried to open an issue about this on tape: https://github.com/substack/tape/issues/508

They suggested me to use `deepLooseEqual` but last time I tried seems also thaat wasn't working

Reply:
Ahah i can imagine, you were more brave than me by digging into tape.
I just tried `deepLooseEqual` and it seems to work.

Reply:
Good news, maybe in last year they have fixed something :laughing: I was quite sure it wasn't working when I tried, If you check all other aedes repos need this fix...